src/Codec/CBOR/Cuddle/CDDL/Resolve.hs

```diff
@@ -2,7 +2,6 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DerivingVia #-}
 {-# LANGUAGE LambdaCase #-}
--- {-# LANGUAGE OverloadedRecordDot #-}
```

Comment:
Reason for commenting rather than deleting this?

Reply:
I deleted a commented pragma, I didn't comment it myself.